dynawo/3rdParty/libiidm/extensions/activeSeason/xml/include/internals/config.h.in

```diff
@@ -19,16 +19,4 @@
 
 #include <IIDM/extensions/activeSeason/xml/version.h>
 
-//cmake generated constants.
-
-#ifndef IIDM_TESTING
-//release builds use xsd in installation path
-#define IIDM_EXT_ACTIVESEASON_XML_XSD_PATH "@CMAKE_INSTALL_PREFIX@/@IIDM_SHARE_PATH@/xsd/"
```

Comment:
The removal of the recompilation of iidm library in the deploy should be included in this commit. To make sure it works, you need to delete the build folder from dynawo and try to compile dynawo-rte and nrt.

Reply:
Done